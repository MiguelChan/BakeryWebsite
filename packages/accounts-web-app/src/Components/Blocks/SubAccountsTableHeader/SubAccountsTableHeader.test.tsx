import React from 'react';
import {
  RenderResult,
  render,
} from '@testing-library/react';
import {
  Table,
} from '@mui/material';
import {
  SubAccountsTableHeader,
} from './SubAccountsTableHeader';

describe('SubAccountsTableHeader', () => {
  const setupComponent = (): RenderResult => {
    const Component = (
      <Table>
        <SubAccountsTableHeader />
      </Table>
    );

    return render(Component);
  };

  it('Should display the SubAccountsTableHeader', () => {
    const renderResult = setupComponent();
    expect(renderResult.container).toMatchSnapshot();
  });
});
