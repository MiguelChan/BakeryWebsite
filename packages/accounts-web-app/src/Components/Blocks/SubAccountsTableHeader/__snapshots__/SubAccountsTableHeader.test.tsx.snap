// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubAccountsTableHeader Should display the SubAccountsTableHeader 1`] = `
<div>
  <table
    class="MuiTable-root css-rqglhn-MuiTable-root"
  >
    <thead
      class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
    >
      <tr
        class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
      >
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
          scope="col"
        >
          Id de Subcuenta
        </th>
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
          scope="col"
        >
          Nombre de subcuenta
        </th>
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
          scope="col"
        />
      </tr>
    </thead>
  </table>
</div>
`;
